test/Interpreter/keypath.swift

```diff
@@ -43,6 +73,20 @@ public func generic_class_constrained_keypath<U, V>(_ c: V) where V : GenericCon
   print(c[keyPath: kp].text)
 }
 
-// CHECK: Swift.KeyPath<main.GenericController<Swift.Int>, main.MyLabel>
+// CHECK: GenericController<Int>.label
 // CHECK: label
 generic_class_constrained_keypath(GenericController(5))
+
+// CHECK: {{\\Controller\.secondLabel!\.text|\\Controller\.<computed 0x.* \(Optional<MyLabel>\)>!\.<computed 0x.* \(String\)>}}
+print(\Controller.secondLabel!.text)
+
+// CHECK: {{\\Controller\.subscript\(_: String\)|\\Controller\.<computed 0x.* \(String\)>}}
+print(\Controller["abc"])
+// CHECK: \S.a
+print(\S.a)
+// CHECK: {{\\Controller\.subscript\(int: Int str: String _: Int\)|\\Controller\.<computed 0x.* \(Int\)>}}
```

Comment:
Are there missing commas here, or does the output really lack commas separating the parameters (and if so, can they be put in)?

Reply:
No commas. Should be easy to add. 

As you may have noticed, in the original impl there were no type annotations, so I left out the commas in an attempt to duplicate the way unapplied functions are written in the swift compiler (no commas or type annotations, i.e. `self.myFunc(someArg: someOtherArg:)`). Tbh I didn't really think about how it looked now that we have annotations again -- I agree that this would be much better with them added.